tests/e2e/testRunner.js

```diff
@@ -229,7 +236,14 @@ const runTests = async () => {
             progressText = `Suite '${testConfig.name}' [${testIndex + 1}/${numOfTests}], iteration [${i + 1}/${config.RUNS}]\n`;
             testLog.updateText(progressText);
 
-            await restartApp();
+            Logger.log('Killing app...');
+            await killApp('android', config.APP_PACKAGE);
+
+            testLog.updateText(`Coolin down phone 🧊 ${config.SUITE_COOL_DOWN / 1000}s\n`);
+            await sleep(config.SUITE_COOL_DOWN);
```

Comment:
Adding the cool down between booting the app again, had the side-effect of actually causing a cold boot, which increased TTI/bundle load JS times significantly but also stabilized standard deviation.